classes/components/forms/submission/ConfirmSubmission.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * @file classes/components/form/submission/ConfirmSubmission.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class ConfirmSubmission
+ * @ingroup classes_controllers_form
+ *
+ * @brief A preset form for the confirm step in the submission wizard
+ */
+
+namespace PKP\components\forms\submission;
+
+use PKP\components\forms\FieldOptions;
+use PKP\components\forms\FormComponent;
+use PKP\context\Context;
+
+define('FORM_CONFIRM_SUBMISSION', 'confirmSubmission');
+
+class ConfirmSubmission extends FormComponent
```

Comment:
It appears to be possible to make a submission to an inactive section if the section is made inactive after the submission has been started. I'd suggest putting a check for the current section being inactive on the submission confirmation form.

Reply:
Ahh, yeah. These checks are there when creating a new submission but not submitting it. This presents a little bit of a challenge because there is not anywhere to put such an error on the review step. I've filed a separate issue to tackle this so it doesn't hold up this merge: https://github.com/pkp/pkp-lib/issues/8408